### PR TITLE
Add Message Reaction and Actions to Message Definition

### PR DESCRIPTION
**Pull request by geekgonecrazy**

### CODE REVIEW

**Comment by sampaiodiego** [comment]
wut

File: src/messages/IMessageReaction.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Interface which represents a reaction which can be added to a message.
+ */
+export interface IMessageReaction {
+    /** Usernames of the users that reacted *
```
Comment (sampaiodiego) [inline src/messages/IMessageReaction.ts]: unfinished comment 🙈

**Comment by geekgonecrazy** [comment]
oops :)

**Comment by rodrigok** [comment]
Please move this Pull Request to the repository https://github.com/RocketChat/Rocket.Chat.Apps-engine since we moved this code to https://github.com/RocketChat/Rocket.Chat.Apps-engine/tree/master/src/definition

Thanks

**Pull request closed without merging**